src/pages/iou/request/step/IOURequestStepDistance.tsx

```diff
@@ -159,7 +191,47 @@ function IOURequestStepDistance({
         // If a reportID exists in the report object, it's because the user started this flow from using the + button in the composer
         // inside a report. In this case, the participants can be automatically assigned from the report and the user can skip the participants step and go straight
         // to the confirm step.
-        if (report?.reportID) {
+        if (report?.reportID && !ReportUtils.isArchivedRoom(report)) {
```

Comment:
Same as before, I think this needs to be moved to an action https://github.com/Expensify/App/blob/main/README.md#actions However, I think you should make sure with someone who knows the App structure more.

Reply:
All good! https://expensify.slack.com/archives/C03TQ48KC/p1713434903788039?thread_ts=1713359203.763509&cid=C03TQ48KC